### DISCUSSION TITLE
error vue infrastructure logique

### DISCUSSION THREAD

**Discussion by misterdiop** [dbarzin/mercator#983] [Q&A]
![mercator](https://github.com/user-attachments/assets/2edf942c-e2c4-4a88-a473-fa1c560f9f1d)
Hello,
 any idea on this error 500 please?
nothing to migrate for command : php artisan migrate

**Comment by dbarzin** [comment]
It should be fixed now.

**Reply by misterdiop** [reply]
Thank you for your reply.does it mean i have to reinstall?

**Comment by dbarzin** [comment]
See the documentation about it : https://github.com/dbarzin/mercator/blob/master/INSTALL.md#upgrade
